go.mod

```diff
@@ -39,7 +39,7 @@ require (
 	github.com/stretchr/testify v1.8.2
 	github.com/tangzero/inflector v1.0.0
 	github.com/withfig/autocomplete-tools/packages/cobra v1.2.0
-	go.mongodb.org/atlas v0.24.1-0.20230405185704-ad47473b3b87
+	go.mongodb.org/atlas v0.24.1-0.20230414144804-e65b9d86f7e8
```

Comment:
what branch is this tracking? we released 0.25.0 yesterday so this should be now 0.25.0

Reply:
This was from v2clientdev
I guess it was released after I pulled. Updating now